pkg/actions/toolkit/enable_test.go

```diff
@@ -0,0 +1,94 @@
+package toolkit_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	gitops "github.com/weaveworks/eksctl/pkg/actions/toolkit"
+	"github.com/weaveworks/eksctl/pkg/actions/toolkit/fakes"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+var _ = Describe("Gitops", func() {
+	var (
+		fakeClientSet  *fake.Clientset
+		fakeFluxClient *fakes.FakeFluxClient
+		opts           *api.Git
+
+		manager *gitops.Manager
+	)
+
+	BeforeEach(func() {
+		fakeClientSet = fake.NewSimpleClientset()
+		fakeFluxClient = new(fakes.FakeFluxClient)
+		opts = &api.Git{Toolkit: &api.Toolkit{Namespace: "fluxv2"}}
+	})
+
+	JustBeforeEach(func() {
+		manager = &gitops.Manager{
+			K8sClientSet: fakeClientSet,
+			Opts:         opts.Toolkit,
+			FluxClient:   fakeFluxClient,
```

Comment:
big nit: rather than making the fields on the  struct public could you use an `export` pattern? Example: https://github.com/weaveworks/eksctl/blob/master/pkg/drain/export_test.go. This pattern is used a lot in the core library as a way of modify private fields for testing, e.g. https://golang.org/src/net/http/export_test.go